### PR TITLE
Jang

### PR DESCRIPTION
**Pull request by yjang43**
Some part of integration is processed.
Parameters written by user from the program will be sent to backend driver and run in a thread.
Right now, emaSession class needs to be defined more to actually run.

self note: need to check daemon option for backend_driver thread. It is possible the thread does not stop when main program closes and crash the program.

### CODE REVIEW

File: beta/backend_driver.py [hunk 1]
```diff
@@ -21,17 +21,25 @@ def __init__(self):
             'ema': EmaSession,
             # ...
         }
+        self.session_count = 0
 
-    def start_session(self, session):
-        pass
+    @staticmethod
+    def start_session(session):
```
Comment (Jaep0805) [inline beta/backend_driver.py:27]: why do we need a separate function for this?

File: beta/backend_driver.py [hunk 2]
```diff
@@ -21,17 +21,25 @@ def __init__(self):
...
 
     def remove_session(self):
         pass
 
     def create_session(self, class_name, options: List):
         # get Session class that was mapped in algorithm_session_map
         session_class = self.algorithm_session_map[class_name]
-        session = session_class(*options)
+        session_number = self._session_number_generator()
+        session = session_class(session_number, self.order_queue, *options)
```
Comment (Jaep0805) [inline beta/backend_driver.py:37]: Great job here, will fix my function accordingly
might want to add a comment on what the options list includes for an emaSession: (initial investment, currency, short_term, etc)

Comment (Jaep0805) [inline beta/backend_driver.py:37]: If i recall correctly, we wanted the session_number to be a string? Might want to cast it

Comment (yjang43) [reply beta/backend_driver.py:37]: I'm considering other Sessions here as well not only ema. For the record purpose though I will put it.

Comment (yjang43) [reply beta/backend_driver.py:37]: Nice catch! Thanks I will fix it accordingly

**Pull request merged**